federation/docs/api-reference/federation/v1beta1/definitions.html

```diff
@@ -685,7 +685,7 @@ <h3 id="_v1beta1_clusterstatus">v1beta1.ClusterStatus</h3>
 </tr>
 <tr>
 <td class="tableblock halign-left valign-top"><p class="tableblock">zones</p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock">Zones is the list of avaliability zones in which the nodes of the cluster exist, e.g. <em>us-east1-a</em>. These will always be in the same region.</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Zones is the list of availability zones in which the nodes of the cluster exist, e.g. <em>us-east1-a</em>. These will always be in the same region.</p></td>
```

Comment:
This is coming from `federation/apis/federation/types.go`.
Fix that file to fix this